### PR TITLE
Password component karma spec

### PR DESCRIPTION
**Pull request by mohamed2a**
prevent js karma test from being copied

### CODE REVIEW

File: generators/client/files-angular.js [hunk 1]
```diff
@@ -399,8 +399,8 @@ const files = {
             templates: [
                 '_karma.conf.js',
                 'spec/_entry.ts',
+                'spec/app/account/password/_password.component.spec.ts',
                 'spec/app/account/password/_password-strength-bar.component.spec.ts',
-                'spec/app/account/settings/_settings.controller.spec.js',
```
Comment (mohamed2a) [inline generators/client/files-angular.js]: Ok @ramzimaalej, revert this deleted line, and run `mvnw `or `mvn spring-boot:run`. #4890

Comment (ramzimaalej) [inline generators/client/files-angular.js]: @mohamed2a IT works find on my end.

Comment (mohamed2a) [inline generators/client/files-angular.js]: image

**Comment by deepu105** [comment]
guys is this ready to merge?

**Comment by mohamed2a** [comment]
yes @deepu105

**Comment by sendilkumarn** [comment]
still travis fails right ? is it a fix for that ?

**Comment by pascalgrimaud** [comment]
@sendilkumarn : I will rebase my PR #4890 and see if it's fixed

**Comment by deepu105** [comment]
There is a protractor test failing.

Thanks & regards,
Deepu

**Comment by sendilkumarn** [comment]
isn't it a karma test ?

**Comment by deepu105** [comment]
I meant in master

Thanks & regards,
Deepu

**Pull request merged**